paper/sections/materials.tex

```diff
@@ -29,16 +51,26 @@ \subsection{Dataset}
 percentage of all standard amino acids present within a protein, as defined by~\cite{gromiha2010protein}. Also known
 as Monopeptide Composition (MPC).
 \item \textbf{Dipeptide Composition (DPC)}: a feature vector of $400$ values ranging from $0$ -- $100$ indicating the
-percentage of all possible ordered dipeptide pairs present within a protein, as defined by~\cite{gromiha2010protein}.
-\item \textbf{Physico-Chemical Composition (PHC)}: a feature vector of $11$ present/absent binary values corresponding
-to physico-chemical residue classes, including: Aliphatic, Neutral, Aromatic, Hydrophobic, Charged, Positively charged,
+percentage of all possible amino acid pairs present within a protein, as defined by~\cite{gromiha2010protein}.
+\item \textbf{Physico-Chemical Composition (PHC)}: a feature vector of $11$ values corresponding
+to percentage composition of physico-chemical residue classes, including: Aliphatic, Neutral, Aromatic, Hydrophobic, Charged, Positively charged,
 Negatively charged, Polar, Small, Large, and Tiny. 
 \item \textbf{Biochemical Composition (AAindex)}: a feature vector of $49$ physical, chemical, energetic, and
 conformational amino acid properties which have been averaged across all amino acids present within the protein.
-\item \textbf{Position-specific scoring matrix (PSSM) profile}: a feature vector of $400$ min-maxed scaled probabilities
-for the occurrence of each amino acid residue and position combination within the sequence. \GK{Hamid, please verify this sentence}.
+\item \textbf{Position-specific scoring matrix (PSSM) profile}: a feature vector of $400$ min-maxed scaled values 
+being generated by summing up all the rows (scores) in the PSSM profile of a sequence that correspond to the same amino acid.
```

Comment:
@HamidHeyde this edit doesn't tell us the meaning of the rows/scores in the PSSM profile. Aren't they probabilities of finding given sequences at given locations? This sentence should make it explicit what the "score" being normalized is, rather than recursively referencing the PSSM profile for its definition.

Reply:
Thanks! Try to keep it to one sentence, somewhere like what I had written just factually correct :)

Reply:
i'll try my best to keep it short